ion_phys/ions/ca40.py

```diff
@@ -25,55 +25,55 @@ def __init__(self, *, B=None, level_filter=None):
             None we include all levels.
         """
         levels = {
-            ground_level: LevelData(g_J=2.00225664),
-            P12: LevelData(),
-            P32: LevelData(),
-            D32: LevelData(),
-            D52: LevelData()
+            ground_level: LevelData(g_J=2.00225664),  # [?]
```

Comment:
this one shouldn't be a question mark. You can cite my thesis until we publish the g factor measurement properly